sdk/servicebus/service-bus/samples/typescript/gettingStarted/atomManagementApi.ts

```diff
@@ -0,0 +1,65 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+ This sample demonstrates how to carry out management operations using connection string as the means of authenticating to Azure Service Bus. 
+ While this sample shows queue related management operations, you can do the same for Topics, Subscriptions and rules.
+
+*/
+
+import {
+  ServiceBusAtomManagementClient,
+  CreateQueueResponse,
+  GetQueueResponse,
+  ListQueuesResponse
+} from "@azure/service-bus";
+
+async function main(): Promise<void> {
+  const serviceBusAtomManagementClient: ServiceBusAtomManagementClient = new ServiceBusAtomManagementClient(
+    "<insert-connection-string>"
+  );
+
+  const createQueueResult: CreateQueueResponse = await serviceBusAtomManagementClient.createQueue(
+    "testQueue",
+    {
+      enablePartitioning: true,
+      requiresSession: true,
+      lockDuration: "PT1M"
+    }
+  );
+  console.log("Created queue: ", createQueueResult.queueName);
+
+  const getQueueResult: GetQueueResponse = await serviceBusAtomManagementClient.getQueue(
```

Comment:
Suggestion: Should this be renamed to `getQueueDetails` or something similar? Otherwise the user might think that once they got the queue they can start sending messages to it.

Reply:
Same concern would apply for `createQueue` as well, naming and API shape may need more thought and can be updated later possibly

Reply:
Rename to use `details` suffix where applicable is complete.
Filed https://github.com/Azure/azure-sdk-for-js/issues/5914 for reconsidering other API and refactoring